docs/user_guides/fs/provenance/provenance.md

```diff
@@ -2,11 +2,64 @@
 
-Hopsworks feature store allows users to track provenance (lineage) between feature groups, feature views and training dataset. Tracking lineage allows users to determine where/if a feature group is being used. You can track if feature groups are being used to create additional (derived) feature groups or feature views. 
+Hopsworks feature store allows users to track provenance (lineage) between storage connectors, feature groups, feature views, training datasets and models. Tracking lineage allows users to determine where/if a feature group is being used. You can track if feature groups are being used to create additional (derived) feature groups or feature views.
 
 You can interact with the provenance graph using the UI and the APIs.
 
-## Step 1: Feature group lineage
+## Step 1: Storage connector lineage
+
+The relationship between storage connectors and feature groups is captured automatically when you create an external feature group. You can inspect the relationship between storage connectors and feature groups using the APIs.
+
+=== "Python"
+
+    ```python
+    # Retrieve the storage connector
+    snowflake_sc = fs.get_storage_connector("snowflake_sc")
+
+    # Create the user profiles feature group
+    user_profiles_fg = fs.create_external_feature_group(
+        name="user_profiles",
+        version=1,
+        storage_connector=snowflake_sc,
+        query="SELECT * FROM USER_PROFILES"
+    )
+    user_profiles_fg.save()
+    ```
+
+### Using the APIs
+
+Starting from a feature group metadata object, you can traverse upstream the provenance graph to retrieve the metadata objects of the storage connectors that are part of the feature group. To do so, you can use the [get_storage_connector_provenance](https://docs.hopsworks.ai/feature-store-api/{{{ hopsworks_version }}}/generated/api/feature_group_api/#get_storage_connector_provenance) method.
+
+=== "Python"
+
+    ```python
+    lineage = user_profiles_fg.get_storage_connector_provenance()
+
+    # List all accessible parent storage connectors
+    lineage.accessible
+
+    # List all deleted parent storage connectors
+    lineage.deleted
+
+    # List all the inaccessible parent storage connectors
+    lineage.inaccessible
+    ```
+
+To traverse the provenance graph in the opposite direction (i.e. from the storage connector to the feature group), you can use the [get_feature_groups_provenance](https://docs.hopsworks.ai/feature-store-api/{{{ hopsworks_version }}}/generated/api/storage_connector_api/#get_feature_groups_provenance) method. When navigating the provenance graph downstream, the `deleted` feature groups are not tracked by provenance, as such, the `deleted` property will always return an empty list.
+
+=== "Python"
+
+    ```python
+    lineage = snowflake_sc.get_feature_groups_provenance()
+
+    # List all accessible downstream feature groups
+    lineage.accessible
+
+    # List all the inaccessible downstream feature groups
+    lineage.inaccessible
+    ```
+
```

Comment:
Can you also add the `get_storage_connector` method.
The expectation is that an external feature group has a accessible parent storage connector that this method will return. This method returns a usable StorageConnector object if it exists and is accessible.

Reply:
also add the `get_feature_groups` method and specify that it return the list of accessible feature groups.